speak/voice.py

```diff
@@ -110,7 +111,24 @@ def __cmp__(self, other):
 
 
 def _init_voice_cache():
-    for language, name, dialect in espeak.voices():
+    speech_manager = speech.get_speech_manager()
+    all_voices = speech_manager.get_all_voices()
+    out = []  # Voices that produce sound only
+
+    for lang_code, name in all_voices.items():
+        if(name in ('en-rhotic', 'english_rp', 'english_wmids')):
+            # these noices do not produce sound
```

Comment:
Typo, noices should be voices?  Do they really not produce sound?

Reply:
Yes, indeed.
I have changed it.

Reply:
I haven't been able to check whether the voices really do not produce sound. I shall try to test it

Reply:
Thanks.  You added this code in the commit, so I thought you already knew.  If you got it from somewhere else, tell me?